### PR TITLE
Fix intellisense

### PR DESCRIPTION
**Pull request by ocallesp**

### CODE REVIEW

**Comment by ocallesp** [comment]
cc @colombod

**Comment by jonsequitur** [comment]
Can get we work together on adding test coverage for this?

**Comment by colombod** [comment]
> Can get we work together on adding test coverage for this?

Tests are there, they are integration tests.

File: src/microsoft-trydotnet-editor/src/ProjectKernelWithWASMRunner.ts [hunk 1]
```diff
@@ -219,7 +204,7 @@ export class ProjectKernelWithWASMRunner extends projectKernel.ProjectKernel {
   }
 
   private deriveToken(originalCommand: polyglotNotebooks.KernelCommandEnvelope) {
```
Comment (colombod) [inline src/microsoft-trydotnet-editor/src/ProjectKernelWithWASMRunner.ts:206]: Is this still used at all? This method should be deleted as we need not to manipulate command tokens.

**Review by colombod — CHANGES_REQUESTED** [review]

**Comment by jonsequitur** [comment]
> > Can get we work together on adding test coverage for this?
> 
> Tests are there, they are integration tests.

Do they not run in CI?

File: src/microsoft-trydotnet-editor/src/ProjectKernelWithWASMRunner.ts (hunk 1 above)
Comment (ocallesp) [reply src/microsoft-trydotnet-editor/src/ProjectKernelWithWASMRunner.ts:206]: sure. I'll do that

**Comment by colombod** [comment]
> > > Can get we work together on adding test coverage for this?
> 
> > 
> 
> > Tests are there, they are integration tests.
> 
> 
> 
> Do they not run in CI?

I think they were disabled due to timeouts provisioning playwright on the ci boxes.

**Review by colombod — APPROVED** [review]

File: src/Microsoft.TryDotNet.Tests/CommandExecutionTests.cs [hunk 2]
```diff
@@ -11,7 +11,7 @@ namespace Microsoft.TryDotNet.Tests;
 
 public class CommandExecutionTests
 {
-    [Fact]
+    [Fact(Skip = "Failing in CI")]
```
Comment (ocallesp) [inline src/Microsoft.TryDotNet.Tests/CommandExecutionTests.cs:14]: This is the only tests that deterministically failed in CI
@colombod  @jonsequitur

**Comment by ocallesp** [comment]
Can someone approve this pr ?

**Review by jonsequitur — APPROVED** [review]

**Pull request merged**